### DISCUSSION TITLE
Welcome to docker-transmission-openvpn project discussions!

### DISCUSSION THREAD

**Discussion by haugene** [haugene/docker-transmission-openvpn#1892] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by haugene** [comment]
Hello all :wave: 

So I've just enabled the discussions feature and I hope that this is going to be a great addition to our documentation as a place where the community can propose and discuss features and help each other with getting set up. We've so far relied on a mix of GitHub issues and the use of Gitter chat but none of them have fully solved our needs. Issues tend to be duplicated a lot, and the chat has largely been unmanned and such been a place for much more questions than answers. Hopefully this tool will prove to be a better fit :crossed_fingers: 

A bit about myself (as the vanilla welcome-message :point_up: encourages)... I created this project almost 7 years ago and after a slow start it has attracted more and more users and contributors and it's been a great journey. This project would not be where it is today without the great ideas and development by 200+ contributors - so thanks a lot to all of you :pray:

Over the years my own ability to invest time here has gone up and down. I recently had a daughter and suddenly we're in a down period again. That's why it's been especially awesome that @pkishino has come in and really picked up the mantle. He's been on the contributor list for about 5 years but has really stepped up the game the last couple of years :raised_hands:

So. Keep that in mind when your posting here. We're not heavily staffed and we both live lives and have jobs :smile: 
Any contribution you can do is greatly appreciated. Helping others in the forums, structuring the documentation, cleaning the code or discussing the architecture and feature choices, +++ - all excellent :ok_hand: :1st_place_medal: 

With that said I'll close the Gitter chat and let's go all in on GitHub discussions instead :100:

**Comment by jdubbs4051** [comment]
Hey all...apologies for not knowing enough to find the answer here but hoping someone can point me in the right direction. Looking to confirm if there is a kill switch enabled or part of this docker image.

**Reply by haugene** [reply]
Hey. Well it's a fair question, but it has been asked before. If it's a relevant comment to us starting to use Disussions on GitHub is another matter :wink: Next time you can consider opening a thread in the Q&A section of the discussions:
https://github.com/haugene/docker-transmission-openvpn/discussions/categories/q-a

And to further help you (and anyone else coming after you) to navigate the archives of asked questions you could do the following:

Search for "kill switch" in the existing issues:
https://github.com/haugene/docker-transmission-openvpn/issues?q=is%3Aissue+%22kill+switch%22+

You can't always expect the top result to be the one you're looking for, but in this case it was:
https://github.com/haugene/docker-transmission-openvpn/issues/1752

Which ultimately leads you to this comment:
https://github.com/haugene/docker-transmission-openvpn/issues/196#issuecomment-373678394

There should be several responses from me on similar questions, but that about sums it up.

**Comment by jdubbs4051** [comment]
Apologies but thank you for pointing me in the right direction

**Reply by haugene** [reply]
No worries 👍
